config/repo-values.yaml

```diff
@@ -208,3 +208,6 @@ sync:
       url: https://dragonchain-charts.s3.amazonaws.com
     - name: couchdb
       url: https://apache.github.io/couchdb-helm/
+    - name: cloudwatch-agent
+      url: https://s2504s.github.io/charts/
```

Comment:
This URL doesn't appear to provide a chart repository. Can you please update the URL to a working repository or fix the repository.

Reply:
Okay, NP
but previously URL (https://apache.github.io/couchdb-helm/) looks like uncorrect too

Reply:
I've  updated my Chart Repo
In order to test please do the following:
```
helm repo add s2504s https://s2504s.github.io/charts/
helm search cloudwatch-agent
```